### ISSUE TITLE
Logger crashloop on 0.4.0 (fission-all)

### ISSUE THREAD

**Issue by SaMnCo** [fission/fission#416]
Hello,

Deploying on K8s 1.7.9 (GKE), and using the latest and greatest helm chart in a clean env (deleted all TPR etc before deployment)

```bash
helm install fission-charts/fission-all --namespace fission --values /path/to/values.yaml
```

Everything works ok but the logger enters in a CrashLoop with : 

```bash
$ kubectl logs -n fission logger-1z6jx logger
Usage:
  fission-bundle --controllerPort=<port>
  fission-bundle --routerPort=<port> [--poolmgrUrl=<url>]
  fission-bundle --poolmgrPort=<port> [--namespace=<namespace>] [--fission-namespace=<namespace>]
  fission-bundle --kubewatcher [--routerUrl=<url>]
  fission-bundle --storageServicePort=<port> --filePath=<filePath>
  fission-bundle --builderMgrPort=<port> [--storageSvcUrl=<url>] [--envbuilder-namespace=<namespace>]
  fission-bundle --timer [--routerUrl=<url>]
  fission-bundle --mqt   [--routerUrl=<url>]
``` 

I can see in issues that the logger is going away, so this may mean that the 0.3.1 chart still tries to deploy it but the option --logger disappeared in 0.4.0. 

If you confirm I can do a quick PR for this, no problem. 

++
Sam

**Comment by prithviramesh** [comment]
Before 0.4.0 the logger pod had both a logger container and a fluentd container, in 0.4.0 we've removed the need for the logger container and the logger pod should only have a fluentd container.

Can you post more information about the CrashLoop please? or join the fission slack and message me there

**Comment by SaMnCo** [comment]
OK this is all due to the chart, in which you have both containers (Fluent and initial Fission logger). The logger is not started anymore, so it crashes and the pod enters a crash loop. If you comment out the logger container in the chart everything goes ok. 

This is just a matter of publishing the charts of 0.4.0 instead of keeping them on 0.3.1.

**Comment by soamvasani** [comment]
#462

**Comment by soamvasani** [comment]
fixed by #462
